packages/application-generic/src/usecases/get-feature-flag/get-feature-flag.use-case.ts

```diff
@@ -24,6 +25,26 @@ export class GetFeatureFlag {
     return await this.featureFlagsService.get(key, defaultValue, context);
   }
 
+  async getGlobal<T>(command: GetGlobalFeatureFlagCommand<T>): Promise<T> {
```

Comment:
In the use case we can differentiate now between functionalities that need context discrimination (getWithContext) and the ones that not through `getGlobal`. Happy to rename to something more suitable. But didn't want to call it `getAnonymous` or similar as that's a naming choice of Launch Darkly and I'd like to leave the FeatureFlagsService agnostic to provider. Also considering that feature flags targeted by environmentId or organizationId fall in the _anonymous_ definition (no userId provided). 🤔